guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,42 @@
+[id="security-features_{context}"]
+= Security features in {Project}
+
+When deploying {Project}, consider implementing security features that enhance the overall security of your system.
+{Project} supports multiple security mechanisms, including FIPS mode, `fapolicyd`, and SELinux, to provide additional layers of protection.
+
+Federal Information Processing Standards (FIPS) Mode::
```

Comment:
I'd like a note that you can't enable FIPS on an already installed `{Project}`. Today we have that in our installation guides, but it's better in the planning guide. FIPS mode is neither a prerequisite nor is enabling FIPS part of the installation procedure. Once you have that, you can drop the FIPS mode parts from our installation guides.